webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsManager.js

```diff
@@ -73,48 +73,47 @@ export class SakaiRubricsManager extends RubricsElement {
               title="${this.tr("toggle_site_rubrics")}"
               aria-label="${this.tr("toggle_site_rubrics")}"
               aria-controls="site_rubrics">
-            <span class="collpase-icon fa fa-chevron-down"></span>
+            <span class="collpase-icon fa fa-chevron-down" aria-hidden="true"></span>
           </button>
           <span class="fw-bold">${this.tr("site_rubrics")}</span>
         </div>
 
         <div class="collapse show" id="site_rubrics">
-          <div class="rubric-title-sorting">
+          <div class="rubric-title-sorting bg-light p-3 rounded">
```

Comment:
`bg-light` is not compatible with dark-mode

Reply:
what can i use instead?